en/docs/learn/multi-channel-self-registration-and-account-confirmation.md

```diff
@@ -0,0 +1,404 @@
+# User Self Registration and Account confirmation via Mobile and Email Channels
+
+WSO2 Identity Server (WSO2 IS) supports self-registration to allow users
+set up their account and receive confirmation when the account is
+created.
+
+When a user self-registers, the self-registration process creates the
+user account and locks the account until the user confirms account
+ownership via a confirmation mail or a SMS One Time Password(OTP) that WSO2 IS sends.
+
+
+
+If a user does not confirm account ownership before the specified expiry
+period, the user account is locked assuming that the expired account is
+not owned by the creator. If necessary, the system administrator can
+delete such accounts to manage resources better.
+
+!!! Note
+    This feature is only available via the Self Registration and Account confirmation REST APis. 
+    Currently, WSO2 IS does not support this feature via the End User Dashboard.
+
+The following sections walk you through configuring and trying out
+self-registration via SMS and Email Channels.
+    
+??? Warning "If you have migrated from a previous IS version"
+    If you have migrated from a previous IS version, ensure that the `IdentityMgtEventListener` with the
+    `         orderId=50        ` is set to **false** and that the Identity
+    Listeners with `         orderId=95        ` and
+    `         orderId=97        ` are set to **true** in the
+    `         <IS_HOME>/repository/conf/deployment.toml        ` file.
+   
+    !!! Note 
+        If there is no such entries for `event.default_listener.xxx`, in `deployment.toml`, 
+        no need to configure the following.
+    
```

Comment:
Shall we change this to 

"You can skip this step if there are no entries for `event.default_listener.xxx`, in  the `deployment.toml`file."

Reply:
Fixed: 030f591d9fd2591616bce502af05234321811549